tests/test_cli.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `mlbench_core.cli` package."""
+
+import pytest
+from mlbench_core.cli import cli_group
+from google.auth.exceptions import DefaultCredentialsError
+
+# import click
+
+create_gcloud = cli_group.commands["create-cluster"].commands["gcloud"].main
+delete_gcloud = cli_group.commands["delete-cluster"].commands["gcloud"].main
+get_status = cli_group.commands["status"]
+delete_run = cli_group.commands["delete"]
+
+CREATE_GCLOUD_DEFAULTS = {
+    "machine_type": "n1-standard-4",
+    "disk_size": 50,
+    "num_cpus": 1,
+    "num_gpus": 0,
+    "gpu_type": "nvidia-tesla-k80",
+    "zone": "europe-west1-b",
+    "preemptible": False,
+}
+
+
+def get_gcloud_cmd_line(args, option_dict=None):
+    """
+    args: (list) command-line arguments
+    option_dict: (dict) key-value for options line in CREATE_GCLOUD_DEFAULTS
+    """
+
+    flags = {
+        "machine_type": "-t",
+        "disk_size": "-d",
+        "num_cpus": "-c",
+        "num_gpus": "-g",
+        "gpu_type": "--gpu-type",
+        "zone": "-z",
+        "preemptible": "-e",
+        "project": "-p",
+        "custom_value": "-v",
+    }
+
+    args = list(map(str, args))
+
+    def get_array(option):
+        # e.g. option='num_gpus' --> ['-g', 0]
+        if options[option] == True:  # check if flag is boolean
+            return [flags[option]]
+        elif options[option] == False:
+            return []
+        else:
+            return [flags[option], options[option]]
+
+    if option_dict is not None:
+        options = option_dict
+
+        cmd_line_options = [
+            str(elem) for option in options for elem in get_array(option)
+        ]
+
+        return args + cmd_line_options
```

Comment:
```suggestion
        return cmd_line_options + args
```

Not that it matters in our case but ideally options come before arguments